lib/api.js

```diff
@@ -354,11 +350,13 @@ api.declare({
   name: 'terminateInstance',
   title: 'Terminate an instance',
   stability: API.stability.experimental,
-  deferAuth: true,
-  scopes: [
-    ['ec2-manager:manage-instances:<region>:<instanceId>'],
-    ['ec2-manager:manage-resources:<workerType>'],
-  ],
+  scopes: {AnyOf: [
```

Comment:
i find this notation sort of confusing, so I'll have to defer to your judgement as to whether it's correct or not.